services/core-api/app/api/mines/tailings/models/tailings.py

```diff
@@ -32,8 +32,22 @@ def json(self):
         }
 
     @classmethod
-    def create(cls, mine, mine_tailings_storage_facility_name, latitude, longitude, consequence_classification_status_code, has_itrb, tsf_operating_status_code,  add_to_session=True):
-        new_tsf = cls(mine_tailings_storage_facility_name=mine_tailings_storage_facility_name, latitude=latitude, longitude=longitude, consequence_classification_status_code=consequence_classification_status_code, has_itrb=has_itrb, tsf_operating_status_code=tsf_operating_status_code)
+    def create(cls,
```

Comment:
Accidentally formatted this file. Added nothing.

Reply:
In the future, you can `git checkout <file>` to revert that file to the state of the commit HEAD is pointing at (usually the last commit of the current branch) and remove all changes. But there's nothing wrong with consistent formatting :D

Warning: using that on a directory will change all contents of a directory.